compiler/ballerina-parser/src/test/resources/declarations/client-decl/client_decl_source_02.bal

```diff
@@ -0,0 +1 @@
+client "http://www.example.com/apis/myapi.yaml" as myapi
```

Comment:
Let's add at least 3 client declarations with errors in one file. In that way we can test the errors before and after the client declaration.